src/webgpu/shader/execution/expression/call/builtin/select.spec.ts

```diff
@@ -141,7 +161,7 @@ g.test('scalar')
 
     await run(
       t,
-      t.params.component === 'af' ? abstractFloatBuiltin('select') : builtin('select'),
+      dataType[t.params.component as scalarKind].shader_builder,
```

Comment:
nit: surprised `as scalarKind` is needed here and below.

Reply:
Not needed, removed usages.